code/modules/antagonists/blob/overmind.dm

```diff
@@ -257,9 +274,11 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 		return FALSE
 	var/current_health = round((blob_core.get_integrity() / blob_core.max_integrity) * 100)
 	hud_used.healths.maptext = MAPTEXT("<div align='center' valign='middle' style='position:relative; top:0px; left:6px'><font color='#82ed00'>[current_health]%</font></div>")
-	for(var/mob/living/simple_animal/hostile/blob/blobbernaut/blobbernaut in blob_mobs)
-		if(blobbernaut.hud_used && blobbernaut.hud_used.blobpwrdisplay)
-			blobbernaut.hud_used.blobpwrdisplay.maptext = MAPTEXT("<div align='center' valign='middle' style='position:relative; top:0px; left:6px'><font color='#82ed00'>[current_health]%</font></div>")
+	for(var/mob/living/basic/blob_minion/blobbernaut/blobbernaut in blob_mobs)
+		if(!blobbernaut.hud_used || !blobbernaut.hud_used.blobpwrdisplay)
```

Comment:
cache whatever datum `hud_used` resolves to for clarity, especially since we need to access it twice